utils/addons/grass-addons-index.sh

```diff
@@ -92,8 +92,8 @@ generate () {
 <body bgcolor=\"#FFFFFF\">
 <h2>GRASS GIS ${major} Addons Manual pages</h2>
 
-<!-- Generated from: /home/martinl/src/grass_addons/tools/addons/ -->
-<!--       See also: https://github.com/OSGeo/grass-addons/blob/master/tools/addons/README.md -->
+<!-- Generated from: /home/martinl/src/grass_addons/utils/addons/ -->
+<!--       See also: https://github.com/OSGeo/grass-addons/blob/master/utils/addons/README.md -->
```

Comment:
This and other URLs are now quite outdated, but perhaps they don't need to be URLs at all when they point to files in the same repo.

Reply:
The link is actual once this PR is merged.

Reply:
Done in 51b3a84a6612528b879b1619f947afc773f3e79f